Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,501 @@
+%  \nomunit{\,\SI{2.9979e8}{\meter\per\second}}}
+\printnomenclature{}
+
+
+% HELPFUL HINT: If you get the warning ``Command terminated with space.'' when using a \command try placing ``%'' or ``{}'' immediately following the command.
+
+% The sections included here are required. Additional sections and subsections may be added as necessary.
+\section{Introduction}
+\label{sec:introduction}
+  % The introduction is a place to give background and context before diving into the subject matter.
+  % Establish context for the work you are about to propose and the main ideas of the proposition itself.
+
+\IEEEPARstart{T}{he} objective of this project is create a cold gas thruster. The CGT will allow the team to gain experience in designing, building,
+and testing propulsion systems. The type of nozzle that will be tested is a minimum length supersonic nozzle. There is ambient pressure, thus the
+nozzle will be optimized for ambient conditions.
+
+\section{Primary Objective}
+\label{sec:primary-obj}
+  % At the end of the day, whether the project ``succeeds'' or ``fails'' is judged against the objectives it sought to meet.
+  % Note that results that contradict expectations/hypotheses are not failures if the scientific \& engineering methods are followed along the way.
+  % Sometimes our expectations are wrong and that can be just as successful as getting data we thought we'd see.
+  % What matters are what questions you intend to answer.
+  % This is the main purpose or main goal the project hopes to achieve.
+The primary objective of this project is to create a cold gas thruster for on-ground experimentation. The goal for the data is to be able to design a nozzle for a small propulsion system, where boundary
+layer effects are dominant, but also to achieve an specific impulse of 50s. Using the thrust over time curves, the specific impulse can be
+calculated numerically using MATLAB\@. The data will be transferred and analyzed there using the following equations as a basis\cite{SPAD}.
+
+
+\begin{equation} \label{total-impulse}
+  I_t    = \int_{t_i}^{t_f} F_{thrust} dt
+\end{equation}
+
+\begin{figure}[ht]
+  \includegraphics[width=\linewidth]{figs/cold-gas-thruster.png}
+  \caption{The Solidworks Assembly of the Spring 2017 design of the SPEX cold gas thruster. The piping from the solenoid to the nozzle is omitted for clarity.}
+\label{fig:lifecycle}
+\end{figure}
+
+\begin{equation} \label{specific-impulse}
+  I_s = \frac{I_t}{m_p g_0}
+\end{equation}
+
+Data error found from past experimentation was 5\% of the thrust. This error is due to the quality of the load cell. It has an inherent inaccuracy. This
+amount of error is acceptable. The performance analysis will not be detrimentally affected. The risk mitigation includes zeroing the load cell just before the test.
+This will erase the effects caused by initial weight placed on the load cell. This initial weight comes from the aluminum sliding fixture which holds the nozzle
```

Comment:
The second sentence is redundant and can be removed. Most of this section could be reworded to indicate the percentage of error expected and an explanation of why that error will not interfere with test results.